src/ResourceManager/EventGrid/Commands.EventGrid.Test/ScenarioTests/TopicTests.cs

```diff
@@ -35,13 +35,20 @@ public void EventGrid_TopicsCreateGetAndDelete()
             EventGridController.NewInstance.RunPsTest("TopicTests");
         }
 
-        [Fact]
+        [Fact(Skip = "Recording blocked on the list subscriptions error under investigation by Azure PowerShell team.")]
```

Comment:
@kalyanaj have you tried to re-record this test recently? I believe others having the same issue you saw have successfully recorded their tests